source/indexes/multikey-index.txt

```diff
@@ -0,0 +1,85 @@
+.. _php-multikey-index:
+
+================
+Multikey Indexes
+================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: index, query, optimization, efficiency
+
+Overview
+--------
+
+**Multikey indexes** are indexes that improve the performance of queries
+on array-valued fields. You can create a multikey index on a collection
+by using the ``MongoDB\Collection::createIndex()`` method and the same
+syntax that you use to create single field or compound indexes.
```

Comment:
S: Might be helpful to link to the single field and compound index pages as you mention them here